ui/app/src/components/LegacySiteDashboard/LegacyApprovedScheduledDashlet/ApprovedScheduledDashlet.tsx

```diff
@@ -98,6 +99,7 @@ export function ApprovedScheduledDashlet() {
     [isAllChecked, selectedLookup, state.itemsLookup]
   );
   const locale = useLocale();
+  const { itemsByPath } = useDetailedItems(Object.keys(selectedLookup));
```

Comment:
Can we get by with SandboxItems here, in all these cases? Detailed items are more expensive.